core/bootstrap/build.gradle.kts

```diff
@@ -26,8 +26,8 @@ dependencies {
 
 publishing {
     publications {
-        create<MavenPublication>("core.bootstrap") {
-            artifactId = "core.bootstrap"
+        create<MavenPublication>("core-bootstrap") {
```

Comment:
Why `bootstrap` and `contract` mantained the `core-`prefix and the other core modules did not?

Reply:
we can remove it from there.